front-end/src/app/shared/services/login.service.ts

```diff
@@ -58,11 +60,15 @@ export class LoginService extends DestroyerComponent {
           window.location.href = environment.loginDotGovLogoutUrl;
         }
       }
-      this.store.dispatch(setCommitteeAccountDetailsAction({ payload: new CommitteeAccount() }));
```

Comment:
Dispatching the userLoggedOutAction now triggers new resolvers in the store for committeeAccount and activeReport